### PR TITLE
Alpha Banner

### PR DESCRIPTION
**Pull request by nmakuch**
### 1) Creating an Alpha Banner component that is a part of the `base.pug` layout

| Desktop | 
|---------|
| Screen Shot 2019-07-24 at 13 57 49    | 

| Mobile |
|--------|
| Screen Shot 2019-07-24 at 13 58 18    |

#### Breakdown
- Designing alpha banner component
- Converting design to html 
- Mobile queries
- French translations
- Css styling

### 2) Aligning french toggle with fip

| Before | After |
|--------|-------|
|   Screen Shot 2019-07-24 at 14 03 01     |   Screen Shot 2019-07-24 at 14 02 17    |

#### Breakdown
- Grouping fip and french toggle into same container
- Applying flexbox properties for alignment
- Mobile queries (`flex: row` on desktop >>> `flex:column` on mobile)

### CODE REVIEW

File: views/base.pug [hunk 1]
```diff
@@ -28,15 +28,21 @@ html(lang=getLocale() === 'fr' ? 'fr' : 'en')
   body
     .container
       header
+        .phase-banner
+          div
+            span ALPHA
+            span This is a new service – your 
+              a(href='#') #{__('feedback')}
```
Comment (pcraig3) [inline views/base.pug]: If we're putting in a link, we should actually have it go somewhere. (maybe a google form?)

Comment (nmakuch) [reply views/base.pug]: removed link and updated to "This site will change as we test ideas."

File: public/scss/components/_header.scss [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-  padding: $space-xs 0 0 0;
+  padding: 0 0 0 0;
+
+  button {
+    padding: 0;
+  }
 
   @include sm {
-    padding: $space-lg 0 $space-xl 0;
+    padding: 0 0 $space-xl 0;
```
Comment (pcraig3) [inline public/scss/components/_header.scss]: This could be simplified to `padding-bottom: space xl`

File: public/scss/_base.scss [hunk 3]
```diff
@@ -76,6 +76,62 @@ li:not(:last-of-type) {
...
+  display: flex;
+
+  @include sm {
+    justify-content: space-between;
+    flex-direction: row;
+  }
+}
+
+.phase-banner {
+  background: #F4F4F4;
```
Comment (pcraig3) [inline public/scss/_base.scss]: Would one of our other colours work? If not, it's fine. If it's just in this one place, we're probably okay.

File: public/scss/_base.scss [hunk 4]
```diff
@@ -76,6 +76,62 @@ li:not(:last-of-type) {
   @extend %page-container;
 }
 
+.fip-container {
```
Comment (pcraig3) [inline public/scss/_base.scss]: Actually, can you make this an extra file? We should try to keep base as small as reasonable.

File: public/styles.scss [hunk 5]
```diff
@@ -20,3 +20,5 @@
 @import './scss/components/_multiple-choice';
 @import './scss/components/_banners';
 @import './scss/components/_breakdown-table';
+@import './scss/components/_fip';
+@import './scss/components/_phase-banner';
```
Comment (pcraig3) [inline public/styles.scss]: Could have put them both in one file if you wanted, but this works too 👍

**Pull request merged**